network-api/networkapi/wagtailpages/tests/test_utils.py

```diff
@@ -17,15 +18,35 @@
 )
 from networkapi.wagtailpages.factory import blog as blog_factories
 from networkapi.wagtailpages.factory.profiles import ProfileFactory
-from networkapi.wagtailpages.pagemodels.blog.blog import BlogAuthors
+from networkapi.wagtailpages.pagemodels.blog.blog import BlogAuthors, BlogPage
 from networkapi.wagtailpages.pagemodels.profiles import Profile
 from networkapi.wagtailpages.utils import (
     create_wagtail_image,
     get_blog_authors,
+    get_content_related_by_tag,
     localize_queryset,
 )
 
 
+class TestGetContentRelatedByTag(TestCase):
+    def test_single_page_has_same_tag(self):
+        # Not using the factories here because that was throwing errors due to missing images for some reason.
```

Comment:
I guess this is a reason for using minimal factories that only set required fields with just as much as needed.